src/events/governance.ts

```diff
@@ -54,7 +45,7 @@ export async function getGovernanceEvents(
   fromBlockNumber: bigint,
   toBlockNumber: bigint,
 ) {
-  const logs = await strategicGetLogs({
+  return await strategicGetLogs({
```

Comment:
fine to remove the assignment ( i did it so i can easier debug ), but then also return the unnecessary await i guess (same in the other files)